torch/__init__.py

```diff
@@ -2499,6 +2502,13 @@ def compile(
         in the function that it will optimize. If True, then we require that the entire function be
         capturable into a single graph. If this is not possible (that is, if there are graph breaks),
         then this will raise an error.
+       error_on_graph_break (bool): If `fullgraph` is set, then this arg does nothing
```

Comment:
once we have some comprehensive documentation on this, might be worth linking to an example that shows when users would want to use this setting